utils/data-generator/main.go

```
@@ -0,0 +1,159 @@
+	"strings"
+	"time"
+
+	"github.com/brianvoe/gofakeit/v7"
+)
+
+func main() {
+	numProjects := flag.Int("count", 5, "Number of test projects to create")
+	baseDir := flag.String("base", ".", "Base directory where project directories will be created")
+	flag.Parse()
+
+	gofakeit.Seed(time.Now().UnixNano())
```

golangci-lint (1.64.8)

Error return value of `gofakeit.Seed` is not checked

(errcheck)